README.md

```diff
@@ -2,11 +2,58 @@ API-Beheer
+> - _Indien er inderdaad sprake is van meerdere standaarden dan kun je denk ik stellen dat je standaarden kunt clusteren. Moeten we daarvoor in GEMMA Online voorzieningen treffen?_
+
+Het dient daarnaast tevens als aanzet om te komen tot een generieke methode voor het in beheer nemen en uitvoeren van dat beheer van andere (nog te ontwikkelen) Open API standaarden.
+De inhoud van deze repository zal op een later moment mogelijk worden opgenomen in een repositroy waarin het beheer van Open API standaarden of van staandarden in het algemeen worden beschreven.
+
+**Visie API-Beheer** 
+
+Het beheer van Open API standaarden waarborgt de ondersteuning van de standaard nadat deze is ontwikkeld.
+Zo worden er vragen beantwoord, problemen met de standaard vastgelegd, vindt er doorontwikkeling plaats, etc... 
+
+Bij het beheer van een Open API standaard onderkennen we 2 fases en elke fase kent een aantal aspecten:
+* In beheername fase.
+  De Open API standaard moet voldoen aan een aantal criteria voordat het in beheer genomen kan worden.
+  Deze worden in deze fase gecheckt en aan de hand van die check volgt de in beheername tenzij er niet aan de
+  criteria wordt voldaan. Idealiter hebben de ontwikkelaars van een standaard de criteria natuurlijk al tijdens de
+  ontwikkeling van de standaard in het oog. Daarnaast zijn de beheerders stakeholders bij de ontwikkeling van een standaard 
+  en houden zij dit aspect ook goed in de gaten. Dit zou dus slechts een formele slag moeten zijn.
+
+  Na acceptatie van de standaard volgt de overdracht van de standaard naar de beheerders.
+
+>   _**Nog te beantwoorden vragen:**_ 
+>  - _is bij de overdracht een standaard al gepubliceerd?_
+>  - _is bij de overdracht het door de leveranciers en gemeenten te gebruiken kanaal al ingericht?_
+>  - _is bij de overdracht de standaard al geconfigureerd in de testvoorziening?_
+>  - _het inrichten van een communicatiekanaal waarop leveranciers en gemeenten hun vragen en problemen kunnen stellen moet voor elke standaard op dezelfde wijze gebeuren. Dus bijv. allemaal op dezelfde locatie, of allemaal binnen de eigen GitHub repository, of nog een andere variant zolang het maar generiek is. Aangezien je n.m.m. niet bij ontwikkeling een ander kanaal wil inrichten dan bij beheer moet dit bij de start van de ontwikkeling van een nieuwe standaard al goed worden gedaan. Dit is dus een beheercriteria dat al speelt bij de aanvang van een ontwikkelproject. Welke criteria kunnen we nog meer onderkennen waarvoor dat geldt?_
+>  - _..._
+
+* Beheerfase.
+  Dit is de fase waarin een standaard daadwerkelijk in beheer is genomen
+  - aspecten die betrekking hebben op de werkzaamheden van de beheerders.
+    + over welke vaardigheden en kennis dienen beheerders te beschikken?
```

Comment:
Dit wordt (mede) bepaald door de taken de de beheerders hebben. Wel/niet zelf aanpassen van de referentie implementatie, genereren van API's uit referentie implementatie en/of een UML model, etc.